pkg/utils/project/project.go

```diff
@@ -30,3 +31,10 @@ func RelativeToRoot(path string) string {
 	manifestsRoot := filepath.Join(filepath.Dir(file), "..", "..", "..")
 	return filepath.Join(manifestsRoot, path)
 }
+
+func GetReleaseVersion() string {
+	if strings.Contains(Version, "-") {
```

Comment:
Debated on whether I should actually do this.  I'm guessing we can't rely on `-` to be in every non-production release.

Reply:
I'm advocating for using karpenter.sh/docs for latest rather than this, although I think this is a fair assumption.

Reply:
I just pushed an update to use the karpenter.sh/docs site instead.